spec/index.bs

```diff
@@ -762,36 +769,74 @@ enum GPUCompareFunction {
 Resource Binding {#binding}
 ===========================
 
-## GPUPipelineLayout ## {#pipeline-layout}
+## GPUBindGroupLayout ## {#bind-group-layout}
+
+A {{GPUBindGroupLayout}} defines an interface between a set of resources bound in a {{GPUBindGroup}} and their accessibility in shader stages.
 
 <script type=idl>
-interface GPUPipelineLayout : GPUObjectBase {
+interface GPUBindGroupLayout : GPUObjectBase {
 };
 </script>
 
-### Creation ### {#pipeline-layout-creation}
+A {{GPUBindGroupLayout}} object has the following internal slots:
 
-<script type=idl>
-dictionary GPUPipelineLayoutDescriptor : GPUObjectDescriptorBase {
-    required sequence<GPUBindGroupLayout> bindGroupLayouts;
-};
-</script>
+<dl dfn-type=attribute dfn-for="GPUBindGroupLayout">
+	: <dfn>\[[bindingsMap]]</dfn> of a Map of `unsigned long` {{GPUBindGroupLayoutBinding/binding}} keys to <dfn dfn for ="layout binding">layout binding</dfn> values.
+	::
+        The [=layout binding=] objects this {{GPUBindGroupLayout}} describes.
 
-## GPUBindGroupLayout ## {#bind-group-layout}
+</dl>
 
-<script type=idl>
-interface GPUBindGroupLayout : GPUObjectBase {
-};
-</script>
+An internal [=layout binding=] object has the following slots:
+
+<dl dfn-type=attribute dfn-for="layout binding">
+    : <dfn>\[[binding]]</dfn> of type unsigned long.
+    ::
+        The unique identifier for this [=layout binding=] in shader stages.
```

Comment:
would be good to define the scope of this uniqueness, e.g. "unique within this bind group layout"